### PR TITLE
Add java checking methods[wip]

### PR DESCRIPTION
**Pull request by DeinFreund**
This is just the methods without using them as I don't know the protocol between Chobby<->Wrapper

### CODE REVIEW

File: ChobbyLauncher/Chobbyla.cs [hunk 1]
```diff
@@ -161,6 +161,11 @@ private dynamic ExtractEngineFromLua(PackageDownloader.Version ver)
 
         private bool CheckForJava()
         {
+            int p = (int)Environment.OSVersion.Platform;
+            bool isLinux = (p == 4) || (p == 6) || (p == 128);
```
Comment (sprunk) [inline ChobbyLauncher/Chobbyla.cs]: magic -> named consts?

**Comment by DeinFreund** [comment]
This only works for windows where spring is always 32 bit for now. :/

I'm not sure if it's even worth trying on Linux because Spring can be really picky with the java.

File: ChobbyLauncher/Chobbyla.cs (hunk 1 above)
Comment (DeinFreund) [inline ChobbyLauncher/Chobbyla.cs]: http://www.mono-project.com/docs/faq/technical/

That's the recommended way of checking.. But I can name them if you want.

Comment (sprunk) [inline ChobbyLauncher/Chobbyla.cs]: I don't want.

Comment (DeinFreund) [inline ChobbyLauncher/Chobbyla.cs]: Ok, because it's not going to make this any less magical to me.

**Comment by Licho1** [comment]
will go to engine

**Comment by DeinFreund** [comment]
Doesn't seem

**Pull request closed without merging**